server/src/test/java/org/opensearch/rest/BaseRestHandlerTests.java

```diff
@@ -337,4 +343,16 @@ public String getName() {
         }
     }
 
+    public void testReplaceRoutesMethod() throws Exception {
+        List<Route> routes = Arrays.asList(new RestHandler.Route(Method.GET, "/path/test"),
```

Comment:
nit: either use `new Route()` (preferable) or `List<RestHandler.Route>` (mixing both is a bad style).

Reply:
Done